server/src/main/kotlin/com/kamelia/hedera/core/Response.kt

```diff
@@ -58,9 +58,14 @@ open class Response<out T>(
         }
     }
 
+    suspend fun ifSuccess(
+        onSuccess: suspend (ResultData<out T>) -> Unit
+    ) = ifSuccessOrElse(onSuccess) {}
```

Comment:
just put the lambda in the parenthesis (you can name the param so that IJ doesnt tell you to do otherwise)